ui/src/pages/backoffice/Document/DocumentDetails/components/DocumentRelations/RelationMultipartMonograph/RelationMultipartModal.js

```diff
@@ -0,0 +1,197 @@
+
+    const pid = this.props.documentDetails.pid;
+    this.props.createRelations(pid, [newRelation]);
+    this.setState({ isLoading: false });
+    this.toggle();
+  };
+
+  /* see HitsSearch.js -> ResultRenderer */
+  serializeSeries = series => ({
+    id: series.metadata.pid,
+    key: series.metadata.pid,
+    title: series.metadata.title,
+    description: series.metadata.mode_of_issuance,
+    extra: `Series #${series.metadata.pid}`,
+    metadata: series.metadata,
+  });
+
+  onSelectResult = result => {
+    const { id } = result;
+    const { recordType, relationType } = this.props;
+    const relatedId = `${id}-${recordType}-${relationType}`;
+
+    result.id = relatedId;
+    result.key = relatedId;
+    this.setState({ selection: result });
+  };
+
+  render() {
+    const { disabled, documentDetails } = this.props;
+    return (
+      <Modal
+        id="es-selector-modal"
+        trigger={
+          <Button
+            disabled={disabled}
+            className="edit-related"
+            icon
+            labelPosition="left"
+            positive
+            onClick={this.toggle}
+          >
+            <Icon name="add" />
+            Add multipart
+          </Button>
+        }
+        open={this.state.visible}
+        centered={true}
+        onClose={this.toggle}
+      >
+        <Modal.Header>Attach document to a multipart monograph</Modal.Header>
+        <Modal.Content>
+          <Container textAlign="center">
+            <Form>
+              <Form.Group>
+                <Container className="spaced">
+                  <HitsSearch
+                    query={seriesApi.multipartMonographs}
+                    delay={250}
+                    minCharacters={3}
+                    placeholder={'Type to find multipart monograph'}
+                    serializer={this.serializeSeries}
+                    onSelect={this.onSelectResult}
+                    ref={element => (this.searchRef = element)}
+                  />
+                </Container>
+              </Form.Group>
+              You can provide volume index for the document in selected
+              multipart (optional)
+              <br /> <br />
+              <Form.Field inline key="volume">
+                <label>Volume index</label>
+                <Input
+                  name="volume"
+                  type="number"
+                  onChange={(e, { value }) => this.setState({ volume: value })}
+                />
+              </Form.Field>
+            </Form>
+            <SelectedMultipart
+              selection={this.state.selection}
+              currentDocument={documentDetails}
+              volume={this.state.volume}
+            />
+          </Container>
+        </Modal.Content>
+        <Modal.Actions>
+          <Button onClick={() => this.toggle()}>Cancel action</Button>
+          <Button
+            positive
+            loading={this.state.isLoading}
+            disabled={this.state.isLoading}
```

Comment:
It looks to me that this button, is either `loading` or `disabled`, so when is it clickable?

Reply:
when it is not loading

Reply:
My bad I read it like it was negated